lib/UndefinedValue.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+const util = require('util');
+
+/**
+ * Error for undefined values
+ *
+ * @extends Error
+ * @class Errors.UndefinedValue
+ * */
+class UndefinedValue extends Error{
+    /**
+   * @example
+   *
+   *if(!importantPassword){
+       const err = new Errors.Undefined;
+       err.setMessage(
+           'importantPassword',
+           importantPassword
+       );
+       throw err;
+   }
+   console.log(`Dont forget this importantPassword: ${importantPassword}, don't!!!`);
+   *
```

Comment:
console log is confusing feels like it is part of the error. I don't think it's necessary.